### DISCUSSION TITLE
Slack: messaging as bot instead of user

### DISCUSSION THREAD

**Discussion by hialvaro** [windmill-labs/windmill#1622] [Questions] [answered]
I've added the Slack OAuht, but when sending messages (such as approvals) the message is sent from the user who set up the slack resource, is there a way to always send the messages as the Windmill bot?

**Comment by rubenfiszel** [comment, marked as answer]
If you setup the slack oauth: https://docs.windmill.dev/docs/misc/setup_oauth#slack then you can add slack in your workspace settings and you will receive a "slack_bot" resource that you can use to send it as your app

**Reply by hialvaro** [reply]
Awesome, I was connecting the resource via the Resources page instead of doing it from settings. Thanks for your support.
